.github/workflows/main.yaml

```diff
@@ -149,7 +123,6 @@ jobs:
           poetry run coverage run -m pytest --with-slow --with-biolink -m slow -n auto
           poetry run coverage xml
           poetry run coverage report -m
-        shell: bash
```

Comment:
@sierra-moxon - I did not spot this before, but setting the bash shell is important to make failing jobs visible for the Windows runner. I add it back in #2633.